app/src/main/java/ro/code4/deurgenta/ui/splashscreen/SplashScreenActivity.kt

```diff
@@ -33,7 +33,7 @@ class SplashScreenActivity : BaseAnalyticsActivity<SplashScreenViewModel>() {
             progressDialog.dismiss()
             val activity = when {
                 loginStatus.isLoggedIn && loginStatus.onboardingCompleted -> MainActivity::class.java
-                loginStatus.isLoggedIn -> OnboardingActivity::class.java
+                loginStatus.isLoggedIn -> RegisterActivity::class.java
```

Comment:
Why was this change made? if the user is currently logged in, they should be redirected to the onboarding related screens.

Reply:
Sorry, de error was due to a rename  from OnboardingActivity to RegisterActivity while creating RegisterActivity

Reply:
hmm this still shows as RegisterActivity :/

Reply:
fixed